controllers/app.controllers.js

```diff
@@ -1,55 +1,128 @@
-const { getAllTopics, getAllUsernames, getArticleFromDbById, getAllArticles, getCommentsForArticle } = require('../models/app.models');
-
+const {
+  fetchAllTopics,
+  fetchAllUsernames,
+  fetchArticleFromDbById,
+  fetchAllArticles,
+  fetchCommentsForArticle,
+  sendCommentByArticleId,
+  removeCommentById,
+  checkUserExistsByName,
+  updateVotes,
+} = require('../models/app.models');
 
 exports.getTopics = async (req, res, next) => {
-    try {
-        const topics = await getAllTopics();
-        res.status(200).send({topics});
-    } catch(err) {
-        next(err);
-    }
-}
+  try {
+    const topics = await fetchAllTopics();
+    res.status(200).send({ topics });
+  } catch (err) {
+    next(err);
+  }
+};
 
 exports.getUsernames = async (req, res, next) => {
-    try {
-        const usernames = await getAllUsernames();
-        res.status(200).send({usernames});
-    } catch(err) {
-        next(err);
-    }
-}
-
+  try {
+    const usernames = await fetchAllUsernames();
+    res.status(200).send({ usernames });
+  } catch (err) {
+    next(err);
+  }
+};
 
 exports.getArticles = async (req, res, next) => {
     try {
-        const articles = await getAllArticles();
-        res.status(200).send({articles});
-    } catch {
-        next(err);
-    }
-}
+        
+    const { sort_by } = req.query;
+    const { order } = req.query;
+    const { topic } = req.query;
```

Comment:
You can declare all of these in a single line. `const { sort_by, order, topic } = ` etc...